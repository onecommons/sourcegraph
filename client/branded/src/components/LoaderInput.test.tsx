import { render } from '@testing-library/react'

import { Input } from '@sourcegraph/wildcard'

describe('LoaderInput', () => {
    it('should render a loading spinner when loading prop is true', () => {
        expect(render(<Input status="loading" />).asFragment()).toMatchSnapshot()
    })

    it('should not render a loading spinner when loading prop is false', () => {
        expect(render(<Input />).asFragment()).toMatchSnapshot()
    })
})
