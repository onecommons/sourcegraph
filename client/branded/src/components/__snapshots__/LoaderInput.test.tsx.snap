// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderInput should not render a loading spinner when loading prop is false 1`] = `
<DocumentFragment>
  <div
    class="container d-flex loader-input"
  >
    <input
      class="form-control with-invalid-icon"
      type="text"
    />
  </div>
</DocumentFragment>
`;

exports[`LoaderInput should render a loading spinner when loading prop is true 1`] = `
<DocumentFragment>
  <div
    class="container d-flex loader-input"
  >
    <input
      class="form-control with-invalid-icon"
      type="text"
    />
    <div
      class="loadingSpinner spinner"
    />
  </div>
</DocumentFragment>
`;
